김민재/week1/2023.03.01/boj1018/boj1018.py

```diff
@@ -0,0 +1,19 @@
+import sys
+
+M, N = map(int, sys.stdin.readline().split())
+
+board = [sys.stdin.readline().rstrip() for m in range(M)]
+color_check = [[0 for n in range(N)] for m in range(M)]
+
+for m in range(M):
+    for n in range(N):
+        color_check[m][n] += ('W' if (m + n) % 2 == 0 else 'B') != board[m][n]
```

Comment:
파이썬의 장점을 적극 활횽하는 코드같네요 ^&^
저는 파이써닉함을 아직 잘 알지 못하기 때문에...
```suggestion
        color_check[m][n] += (('W' if (m + n) % 2 == 0 else 'B') != board[m][n])
```
저의 조잡한 생각으로는 이렇게 괄호로 한 번 더 감싸는것도 괜찮다고 생각합니다!

Reply:
연산자 우선순위에 따라서 비교연산자가 먼저 처리되기는 하지만 언제나 명시적인것이 헷갈릴 가능성을 줄여주죠 👍